<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-02T12:27:18 IST" name="Default suite" finished-at="2024-06-02T12:28:02 IST" duration-ms="44157">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.Run@558bdf1f]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2024-06-02T12:27:18 IST" name="Default test" finished-at="2024-06-02T12:28:02 IST" duration-ms="44157">
      <class name="TestRunner.Run">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:TestRunner.Run@558bdf1f]" started-at="2024-06-02T12:27:18 IST" name="setUpClass" finished-at="2024-06-02T12:27:20 IST" duration-ms="1487" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@37fef327]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.Run@558bdf1f]" started-at="2024-06-02T12:27:20 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-06-02T12:28:02 IST" duration-ms="42327" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Successful Login with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Verify Login Functionality"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at StepDefination.LoginDef.SetUp(LoginDef.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:TestRunner.Run@558bdf1f]" started-at="2024-06-02T12:28:02 IST" name="tearDownClass" finished-at="2024-06-02T12:28:02 IST" duration-ms="209" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- TestRunner.Run -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
